www/_posts/2020-06-29-cordova-android-9.0.0.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Cordova Android 9.0.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that we have just released `Cordova Android 9.0.0`!  This is one of Cordova's supported platforms for building Android mobile applications.
+
+* [cordova-android@9.0.0](https://www.npmjs.com/package/cordova-android)
+
+## Release Highlights
+
+**To upgrade:**
+
+```bash
+cordova platform remove android
+cordova platform add android@9.0.0
+```
+
+## Release Highlights
+
+* **Added Kotlin Support**
+
+Kotlin is one of the newest statically typed languages for developing Android apps. It is designed to work fully with the existing language, Java.
+
+By default, Cordova has Kotlin disabled but can be enabled with the `GradlePluginKotlinEnabled` `preference` tag.
+
+Additionally, Kotlin’s coding style and version are configurable. By default, Cordova sets the coding style as `official` and uses version `1.3.50`.
+
+Below is an example `config.xml` for enabling and configuring Kotlin.
+
+```xml
+<preference name="GradlePluginKotlinEnabled" value="true" />
+<preference name="GradlePluginKotlinCodeStyle" value="official" />
+<preference name="GradlePluginKotlinVersion" value="1.3.50" />
+```
+
+For plugin developers, it is recommended to ensure that the Kotlin files are placed into the platforms `src/main/kotlin` directory.
+
+* **Added AndroidX Support**
+
+AndroidX is the newest and improved namespace for the Android Support Libraries. The original support libraries are no longer maintained.
+
+It is recommended to use AndroidX so that your app is running the latest support libraries but, by default, Cordova has AndroidX support disabled.
```

Comment:
maybe?
"Cordova has AndroidX support disabled" => "Cordova has AndroidX support disabled for compatibility with existing plugins"